expression/builtin_time.go

```diff
@@ -2516,7 +2516,42 @@ type builtinTimeFormatSig struct {
 // eval evals a builtinTimeFormatSig.
 // See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_time-format
 func (b *builtinTimeFormatSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("TIME_FORMAT")
+	args, err := b.evalArgs(row)
```

Comment:
Thanks. I have a question about the example: what is FieldType.Flen? why bf.tp.Flen = 10?